client/views/msg_preview.tpl

```diff
@@ -5,15 +5,22 @@
 			<img src="{{thumb}}">
 		</a>
 	{{else}}
-		{{#if thumb}}
-			<a class="toggle-thumbnail" href="{{link}}" target="_blank" rel="noopener">
-				<img src="{{thumb}}" class="thumb">
+		{{#equal type "video"}}
```

Comment:
This file should probably start getting abstracted out if we are going to add a bunch of other ones to it. I know you are only adding 2 at this point, but it's going to get too big, I think.

Reply:
Agreed. need to investigate handlebars `{{else if}}`, but I can't figure out how it works with helpers. For a future PR.

Reply:
Yeah, I wasn't meaning for this PR, it's just something we should do before we start adding third party site previews and stuff. The 2 PRs you have up are fine, it's not too big yet, it's just clearly **going** to get too big.